HackIllinois/FlowControllers/HIApplicationStateController.swift

```diff
@@ -50,8 +50,10 @@ class HIApplicationStateController {
         onboardingViewController.shouldDisplayAnimationOnNextAppearance = user == nil
 
         UserDefaults.standard.set(true, forKey: HIConstants.APPLICATION_INSTALLED_KEY)
-        let shouldShowOnboarding = UserDefaults.standard.object(forKey: HIConstants.SHOULD_SHOW_ONBOARDING_KEY) as? Bool ?? true
+//        let shouldShowOnboarding = UserDefaults.standard.object(forKey: HIConstants.SHOULD_SHOW_ONBOARDING_KEY) as? Bool ?? true
```

Comment:
Make sure to change this code to what it originally was. Otherwise, onboarding will be shown every time the app runs.